### PR TITLE
Update configuration-of-price-modes-and-types.md

### PR DESCRIPTION
**Pull request by profuel**
## PR Description

Update examples of the price mode change

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md [hunk 1]
```diff
@@ -80,19 +80,55 @@ You can use these keys in environment configuration:
    		<input type="hidden" name="referrer-url" value="{% raw %}{{{% endraw %} app.request.requestUri {% raw %}}}{% endraw %}" />
    	</form>
    {% raw %}{%{% endraw %} endif {% raw %}%}{% endraw %}
-   ```
+```
+
+
+The switch can happen only if quote have to items.
+
+This is available after the product currency release so you must first follow the steps in [Upgrade the Price module](/docs/pbc/all/price-management/{{site.version}}/base-shop/install-and-upgrade/upgrade-modules/upgrade-the-price-module.html).
```
Comment (andriitserkovnyi) [inline docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md:88]: Is this a prerequisite then?

File: docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md [hunk 2]
```diff
@@ -80,19 +80,55 @@ You can use these keys in environment configuration:
...
+
+## Switching to net prices
+
+To show only NET prices in the catalog, but use gross prices in cart and checkout, override `getDefaultPriceMode` and change to `PRICE_MODE_NET` in the `\Spryker\Shared\Price\PriceConfig` class.
+
+Example of changing the mode for all stores:
+```php
+<?php
 
-5. Place the template you just created to `Pyz/Yves/Price/Theme/default/partial/price_mode_switcher.twig`
```
Comment (profuel) [inline docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md:85]: Why did you delete this?
I cannot verify if this is a needed cleanup.

File: docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md (hunk 1 above)
Comment (profuel) [reply docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md]: it's so old, so I'd even remove this completely. it was needed in 2021

File: docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md [hunk 3]
```diff
@@ -80,19 +80,55 @@ You can use these keys in environment configuration:
    		<input type="hidden" name="referrer-url" value="{% raw %}{{{% endraw %} app.request.requestUri {% raw %}}}{% endraw %}" />
    	</form>
    {% raw %}{%{% endraw %} endif {% raw %}%}{% endraw %}
-   ```
+```
+
+
+The switch can happen only if quote have to items.
```
[the remark this answers is not in the document]
Comment (profuel) [reply docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md]: changed the wording

File: docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md (hunk 2 above)
Comment (andriitserkovnyi) [reply docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md:85]: It's [step 3 now](https://github.com/spryker/spryker-docs/pull/2838/files#diff-138b98dad7df88e223cc63b9b7f014c925f88030187928f7783606c60d3d2948R69)
